doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -249,34 +250,34 @@ This chapter describes the system parameters used by the components of the X-Roa
-| enabled-body-logging-local-producer-subsystems   |                                            | Subsystem-specific overrides for SOAP body logging when soap-body-logging = false.<br/><br/>This parameter defines logging for **local producer** subsystems, that is, our subsystems that produce some service which external clients use.<br/><br/>Comma-separated list of client identifiers for which SOAP body logging is enabled. For example FI/ORG/1710128-9/SUBSYSTEM\_A1, FI/ORG/1710128-9/SUBSYSTEM\_A2 where<br/>-   FI = x-road instance<br/>-   ORG = member class<br/>-   1710128-9 = member code<br/>-   SUBSYSTEM\_A1 = subsystem code<br/><br/>This parameter can only be used on subsystem-level, it is not possible to configure SOAP body logging per member.<br/><br/>If a subsystem has forward slashes “/” in for example subsystem code, those subsystems can’t be configured with this parameter. |
-| enabled-body-logging-remote-producer-subsystems  |                                            | Subsystem-specific overrides for **remote producer** subsystems, that is, remote subsystems that produce services which we use.<br/><br/>Parameter is used when soap-body-logging = false. |
-| disabled-body-logging-local-producer-subsystems  |                                            | Same as enabled-body-logging-local-producer-subsystems, but this parameter is used when soap-body-logging = true. |
-| disabled-body-logging-remote-producer-subsystems |                                            | Same as enabled-body-logging-remote-producer-subsystems, but this parameter is used when soap-body-logging = true. |
-| acceptable-timestamp-failure-period              | 14400                                      | Defines the time period (in seconds) for how long is time-stamping allowed to fail (for whatever reasons) before the message log stops accepting any more messages (and consequently the security server stops accepting requests). Set to 0 to disable this check. The value of this parameter should not be lower than the value of the central server system parameter *timeStampingIntervalSeconds.*     <BR/><BR/>Default for Finnish installation is 18 000 seconds |
-| archive-interval                                 | 0 0 0/6 1/1 \* ? \*                        | CRON expression \[CRON\] defining the interval of archiving the time-stamped messages. |
```

Comment:
Remove text 'Default for Finnish installation is 18 000 seconds'. It is duplicate information as